artifactory/commands/transfer/settings.go

```diff
@@ -0,0 +1,43 @@
+package transfer
+
+import (
+	"errors"
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-cli-core/v2/utils/ioutils"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+	"strconv"
+)
+
+type TransferSettingsCommand struct {
+}
+
+func NewTransferSettingsCommand() *TransferSettingsCommand {
+	return &TransferSettingsCommand{}
+}
+
+func (tst *TransferSettingsCommand) Run() error {
+	var threadsNumberInput string
+	ioutils.ScanFromConsole("Choose the threads number", &threadsNumberInput, "")
```

Comment:
Choose the threads number --> Enter a numeric value for the number of working threads